tests/post_install/conftest.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import requests
+
+from pytest_bdd import given
+from pytest_bdd import when
+
+from utils.helper import Inventory
+
+
+@pytest.fixture(scope="session")
+def inventory_obj(inventory):
+    return Inventory(filename=inventory)
+
+
+@pytest.fixture(scope="session")
+def kubectl_proxy(request, run_services, watcher_getter, kubeconfig):
+    def api_checker():
+        try:
+            response = requests.get('http://127.0.0.1:8001/')
+        except requests.exceptions.RequestException:
+            return False
+        return bool(response)
```

Comment:
What's `bool(response)` supposed to be? Alway `True`? In which case is it `False`?